src/script/interpreter.cpp

```diff
@@ -1126,8 +1166,30 @@ bool TransactionSignatureChecker::CheckSig(const vector<unsigned char>& vchSigIn
     return true;
 }
 
-bool TransactionSignatureChecker::CheckLockTime(const CScriptNum& nLockTime) const
+bool TransactionSignatureChecker::CheckLockTime(const CScriptNum& nLockTime, bool fSequence) const
 {
+    // Relative lock times are supported by comparing the passed
+    // in lock time to the sequence number of the input. All other
+    // logic is the same, all that differs is what we are comparing
+    // the lock time to.
+    int64_t txToLockTime;
+    if (fSequence) {
+        // Fail under all circumstances if the transaction's version
+        // number is not set high enough to enable enforced sequence
+        // number rules.
+        if (txTo->nVersion < 2)
+            return false;
+        // Sequence number must be inverted to convert it into a
+        // relative lock-time.
+        txToLockTime = (int64_t)~txTo->vin[nIn].nSequence;
+        // Sequence numbers under SEQUENCE_THRESHOLD are not consensus
+        // constrained.
```

Comment:
Could this please read:  `Inverted sequence numbers above (or equal to) SEQUENCE_THRESHOLD are not consensus constrained`